src/lwc/geFormRenderer/geFormRendererHelper.js

```diff
@@ -0,0 +1,11 @@
+export function flatten(obj) {
```

Comment:
Could we add a description for this function?

Reply:
Yep, good call, thanks!  Just pushed a commit with a description.